diff --git a/Cargo.toml b/Cargo.toml
index f26fe3e..afc89e3 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -22,6 +22,7 @@ keywords = ["cryptography", "openpgp", "pgp", "encryption", "signing"]
 categories = ["cryptography", "authentication", "email"]
 license = "GPL-2.0-or-later"
 repository = "https://gitlab.com/sequoia-pgp/sequoia-octopus-librnp"
+exclude = ["notes.org", "rust-toolchain"]
 
 [lib]
 crate-type = ["cdylib"]
@@ -61,7 +62,6 @@ default-features = false
 
 [dependencies.rusqlite]
 version = "0.24"
-features = ["bundled"]
 
 [dependencies.sequoia-autocrypt]
 version = "0.23"
