From 6510e5e27937114508925efcf22c5b6fadb2631e Mon Sep 17 00:00:00 2001
From: Fabio Valentini <decathorpe@gmail.com>
Date: Wed, 7 Apr 2021 12:16:03 +0200
Subject: [PATCH] build: drop unavailable ECC functions

---
 build.rs | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/build.rs b/build.rs
index 24e2d42..eee9eb1 100644
--- a/build.rs
+++ b/build.rs
@@ -92,6 +92,8 @@ fn real_main() -> Result<()> {
         .blacklist_function("qgcvt")
         .blacklist_function("qecvt_r")
         .blacklist_function("qfcvt_r")
+        .blacklist_function("nettle_get_secp_192r1")
+        .blacklist_function("nettle_get_secp_224r1")
         .size_t_is_usize(true);
 
     for p in include_paths {
-- 
2.30.2

