From 30eec5ee5e68d815ab9397d4b56caf5bcf6fc33e Mon Sep 17 00:00:00 2001
From: Fabio Valentini <decathorpe@gmail.com>
Date: Thu, 8 Apr 2021 23:20:28 +0200
Subject: [PATCH] split off librnp and make it possible to use a different
 OpenPGP backend

---
 thunderbird.spec | 20 +++++++++++++++++++-
 1 file changed, 19 insertions(+), 1 deletion(-)

diff --git a/thunderbird.spec b/thunderbird.spec
index 0404810..06666a8 100644
--- a/thunderbird.spec
+++ b/thunderbird.spec
@@ -90,7 +90,7 @@ ExcludeArch: s390x
 Summary:        Mozilla Thunderbird mail/newsgroup client
 Name:           thunderbird
 Version:        78.8.1
-Release:        2%{?dist}
+Release:        3%{?dist}
 URL:            http://www.mozilla.org/projects/thunderbird/
 License:        MPLv1.1 or GPLv2+ or LGPLv2+
 Source0:        ftp://ftp.mozilla.org/pub/thunderbird/releases/%{version}%{?pre_version}/source/thunderbird-%{version}%{?pre_version}.source.tar.xz
@@ -206,11 +206,25 @@ BuildRequires:  nasm >= 1.13
 BuildRequires:  icu
 %endif
 
+# require an RNP implementation and prefer the bundled one
+Requires:       thunderbird-librnp
+Suggests:       thunderbird-librnp-rnp
+
 Suggests:       u2f-hidraw-policy
 
 %description
 Mozilla Thunderbird is a standalone mail and newsgroup client.
 
+%package librnp-rnp
+Summary: OpenPGP implementation for Thunderbird based on RNP
+Requires: %{name}
+Provides: thunderbird-librnp
+%description librnp-rnp
+The thunderbird-librnp-rnp package contains an OpenPGP implementation
+based on RNP.
+%files librnp-rnp
+%{mozappdir}/librnp.so
+
 %package wayland
 Summary: Thunderbird Wayland launcher.
 Requires: %{name}
@@ -723,6 +737,7 @@ gtk-update-icon-cache %{_datadir}/icons/hicolor &>/dev/null || :
 %{mozappdir}/thunderbird-bin
 %{mozappdir}/thunderbird
 %{mozappdir}/*.so
+%exclude %{mozappdir}/librnp.so
 %{mozappdir}/platform.ini
 %{mozappdir}/application.ini
 %{mozappdir}/features/*.xpi
@@ -753,6 +768,9 @@ gtk-update-icon-cache %{_datadir}/icons/hicolor &>/dev/null || :
 #===============================================================================
 
 %changelog
+* Thu Apr 08 2021 Fabio Valentini <decathorpe@gmail.com> - 78.8.1-3
+- Split off librnp and make it possible to use a different OpenPGP backend.
+
 * Tue Mar 30 2021 Jonathan Wakely <jwakely@redhat.com> - 78.8.1-2
 - Rebuilt for removed libstdc++ symbol (#1937698)
 
-- 
2.30.2

