--- memsec-0.5.7/Cargo.toml	2020-01-06T15:17:37+00:00
+++ memsec-0.5.7/Cargo.toml	2021-04-08T11:57:29.067784+00:00
@@ -29,16 +29,9 @@
 alloc = ["getrandom"]
 default = ["use_os", "alloc"]
 nightly = []
-use_os = ["libc", "winapi", "mach_o_sys"]
-[target."cfg(any(target_os = \"macos\", target_os = \"ios\"))".dependencies.mach_o_sys]
-version = "0.1"
-optional = true
+use_os = ["libc"]
 [target."cfg(unix)".dependencies.libc]
 version = "0.2"
-optional = true
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3"
-features = ["memoryapi", "sysinfoapi"]
 optional = true
 [badges.appveyor]
 repository = "quininer/memsec"
