--- rusqlite-0.24.2/Cargo.toml	1970-01-01T00:00:00+00:00
+++ rusqlite-0.24.2/Cargo.toml	2021-04-08T17:37:56.828728+00:00
@@ -22,6 +22,8 @@
 categories = ["database"]
 license = "MIT"
 repository = "https://github.com/rusqlite/rusqlite"
+exclude = ["appveyor.yml", "clippy.toml", "codecov.yml", "publish-ghp-docs.sh", "test.csv"]
+
 [package.metadata.docs.rs]
 all-features = false
 default-target = "x86_64-unknown-linux-gnu"
@@ -132,9 +134,6 @@
 backup = ["libsqlite3-sys/min_sqlite_version_3_6_23"]
 blob = ["libsqlite3-sys/min_sqlite_version_3_7_7"]
 buildtime_bindgen = ["libsqlite3-sys/buildtime_bindgen"]
-bundled = ["libsqlite3-sys/bundled", "modern_sqlite"]
-bundled-full = ["array", "backup", "blob", "bundled", "chrono", "collation", "column_decltype", "csvtab", "extra_check", "functions", "hooks", "i128_blob", "limits", "load_extension", "serde_json", "series", "trace", "unlock_notify", "url", "uuid", "vtab", "window"]
-bundled-windows = ["libsqlite3-sys/bundled-windows"]
 collation = []
 column_decltype = []
 csvtab = ["csv", "vtab"]
@@ -145,16 +144,14 @@
 in_gecko = ["modern_sqlite", "libsqlite3-sys/in_gecko"]
 limits = []
 load_extension = []
-modern_sqlite = ["libsqlite3-sys/bundled_bindings"]
+modern_sqlite = []
 series = ["vtab"]
 session = ["libsqlite3-sys/session", "hooks"]
 sqlcipher = ["libsqlite3-sys/sqlcipher"]
 trace = ["libsqlite3-sys/min_sqlite_version_3_6_23"]
 unlock_notify = ["libsqlite3-sys/unlock_notify"]
 vtab = ["libsqlite3-sys/min_sqlite_version_3_7_7", "lazy_static"]
-wasm32-wasi-vfs = ["libsqlite3-sys/wasm32-wasi-vfs"]
 window = ["functions"]
-winsqlite3 = ["libsqlite3-sys/winsqlite3"]
 with-asan = ["libsqlite3-sys/with-asan"]
 [badges.appveyor]
 repository = "rusqlite/rusqlite"
