--- sha1collisiondetection-0.2.3/Cargo.toml	2020-12-15T15:11:13+00:00
+++ sha1collisiondetection-0.2.3/Cargo.toml	2021-04-08T17:49:35.140640+00:00
@@ -22,6 +22,7 @@
 categories = ["cryptography", "no-std"]
 license = "MIT"
 repository = "https://gitlab.com/sequoia-pgp/sha1collisiondetection"
+exclude = ["test", "vs2015", "compile_commands.json", "Makefile", "rust-toolchain", "test-sha1sum-protocol.sh"]
 
 [lib]
 path = "lib/lib.rs"
